client/.eslintrc.js

```diff
@@ -3,10 +3,49 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
     'plugin:prettier/recommended',
   ],
-  plugins: ['@typescript-eslint'],
+  plugins: ['@typescript-eslint', 'import', 'simple-import-sort'],
   rules: {
     'no-console': 'warn',
     'no-undef': 'error',
+    // Rules for auto sort of imports
+    'simple-import-sort/imports': [
+      'error',
+      {
+        groups: [
+          // Side effect imports.
+          ['^\\u0000'],
+          // Packages.
+          // Packages. `react` related packages come first.
+          // Things that start with a letter (or digit or underscore), or
+          // `@` followed by a letter.
+          ['^react', '^@?\\w'],
+          // Root imports
+          // Shared imports should be separate from application imports.
+          ['^(~shared)(/.*|$)'],
+          ['^(~)(/.*|$)'],
+          ['^(~typings)(/.*|$)'],
+          [
```

Comment:
Question: how are these rules generated? I think the config for lines 27-37 may not be applied for our repo as we do not use alias imports of the structure `~assets` in this repository.

Reply:
haha I straight up copied directly from `ts-template`!

I did see these kinda imports in the codebase though (is this what you meant by alias imports?):
<img width="550" alt="image" src="https://user-images.githubusercontent.com/67887489/155930718-d5617fbc-47e0-4050-a8bb-a057cfe20b4d.png">

Reply:
Yes, this repo uses an alias for shared imports (can checkout `tsconfig.paths.json`), which is covered by line 24 of this config. But I think the other lines of config added here from 27-37 will not be used since we don't use alias imports for this repo.

Reply:
To check out why ts-template uses that particular setting, that is because they use alias imports for those folders here https://github.com/opengovsg/ts-template/blob/develop/frontend/tsconfig.paths.json

Reply:
Ah I see your point. OK will remove lines 27-37.